faq.rst

```diff
@@ -0,0 +1,155 @@
+FAQ
+==============
+
+Truth be told, these questions aren't frequently asked. Hopefully the answers are still helpful. 
+
+.. _what-is-ODK:
+
+What is ODK? 
+~~~~~~~~~~~~~~~~~~
+
+Open Data Kit (ODK) is a free and open-source set of tools which help organizations create mobile data collection solutions.
+
+Our `blog <https://opendatakit.org/blog/>`_ blog and `deployments page <https://opendatakit.org/about/deployments/>`_ have good examples of what others have used ODK for. Our `research page <https://opendatakit.org/about/research/>`_ also has videos and slides that explain ODK.
+
+.. _how-use: 
+
+How do I use ODK?
+~~~~~~~~~~~~~~~~~~~~
+
+Please read through the documentation on our `implementer instructions <https://opendatakit.org/use/>`_ and on the `developer wiki <https://github.com/opendatakit/opendatakit/wiki>`_.
+
+If that doesn't help, search the `ODK forum <https://forum.opendatakit.org/>`_ to see if the issue has been answered.
+
+If you don't find an answer after searching, then post your question to the appropriate category.
+
+.. _what-kinds-question:
+
+What kinds of questions do you answer on the mainling list? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our team only answers questions about the `supported tools <https://opendatakit.org/about/tools/>`_. 
```

Comment:
If this is a single paragraph, it should be one line of source code.
If it is three paragraphs, they need blank lines between.